fake_ubersmith/api/methods/client.py

```diff
@@ -169,6 +183,17 @@ def client_cc_delete(self, form_data):
             )
         return response(data=True)
 
+    def _get_all_contacts_response(self, lookup_key, matcher_key, matcher_value):
+        contacts = {
```

Comment:
Do not forget you need to return the primary contact i.e. the client in that method. So you need to get it from the clients and not the contacts. The tests should reflect that as well.